f1newrelic/files/newrelic-infra.yml.jinja

```diff
@@ -1,16 +1,12 @@
-{% from "f1newrelic/map.jinja" import infra_agent_name, project, newrelic_license with context %}
-
 license_key: {{ newrelic_license }}
 display_name: {{ project }}.byf1.dev ({{ infra_agent_name }})
-
 {% if grains.roles is defined %}
 custom_attributes:
-{%- for role in grains.roles %}
-  {% if role == 'utility' %}
+{%- for role in grains.roles -%}
+  {%- if role == 'utility' %}
```

Comment:
Does this just add `role: utility` to utility instances? If so, is there any reason we couldn't inject `grains.get('roles', [])` into the context and simplify the logic to just this?

```sls
{% if 'utility' in roles %}
custom_attributes:
  role: utility
{% endif %}
```